se2_planning_ros/src/OmplReedsSheppPlannerRos.cpp

```diff
@@ -40,29 +40,94 @@ bool OmplReedsSheppPlannerRos::plan() {
 }
 
 bool OmplReedsSheppPlannerRos::planningService(PlanningService::Request& req, PlanningService::Response& res) {
+  // TODO Grid map in state validator is currently initialized to some default values, this check is therefore useless
+  if (!planner_->as<OmplReedsSheppPlanner>()->getStateValidator().isInitialized()) {
```

Comment:
we can avoid this if we move the setStateValidator to the base class. since it is an interface anyway, that is less ugly.

we can then do planner_->getStateValidator().isInitialized().

Reply:
Done in 8c0bc04c133f48fb54ed0649a771297a21539357.